addons/mail/static/src/models/thread/thread.js

```diff
@@ -1252,6 +1270,9 @@ function factory(dependencies) {
         followers: one2many('mail.follower', {
             inverse: 'followedThread',
         }),
+        suggestedPartners: many2many('mail.suggested_partner', {
+            default: []
+        }),
```

Comment:
- no default
- missing docstring for this field
- rename field to `suggestedRecipientInfoList` (name's not great, but info in uncountable in English hence need for suffix "List")